tests/test-addons.py

```diff
@@ -480,3 +481,18 @@ def test_sosivio(self):
         validate_sosivio()
         print("Disabling sosivio")
         microk8s_disable("sosivio")
+
+    @pytest.mark.skipif(
+        platform.machine() != "x86_64",
+        reason="Ondat tests are only relevant in x86 architectures",
+    )
+    def test_ondat(self):
+        """
+        Setup and validates Ondat.
+        """
+        print("Enabling Ondat.")
+        microk8s_enable("ondat")
```

Comment:
Hi @hubvu.

Regarding the test failures, it looks like this expects input from stdin (number of etcd replicas), but it never gets any input in the test case.

For example, I can replicate this by piping /dev/null:

```
root@uk8s:~# microk8s enable ondat < /dev/null
 Enabling CoreDNS for Service Discovery... 
Infer repository core for addon dns
Addon core/dns is already enabled
 Enabling Hostpath Storage for Ondat's etcd Cluster... 
Infer repository core for addon hostpath-storage
Addon core/hostpath-storage is already enabled
 Enabling Helm 3 Package Manager to deploy the Ondat Helm Chart... 
Infer repository core for addon helm3
Addon core/helm3 is already enabled
 Ondat dependency add-ons enabled... 

  *** Ondat 'etcd' Cluster Replicas Sizing Guide ***
    
  - Single-node Cluster (=1 node)   = '1' etcd replica.
  - Multi-node Cluster  (=3 nodes)  = '3' etcd replcias (For Test Environments).
  - Multi-node Cluster  (>=5 nodes) = '5' etcd replcias (For Prod Environments).

root@uk8s:~# echo $?
1
```

Can you allow the enable script to run in a non-interactive way? E.g. falling back to a number of etcd nodes by default if reading fails, or specifying the number of etcd replicas using a command line argument?

Thank you!

FYI; the MicroK8s 1.26 release is in 2 weeks, we would like to make sure the addon gets included, so reach out if you need help.

Reply:
@neoaggelos thanks for the feedback on this - and yes I can make it run in a non interactive way, let me think of a decent way of deploying to a single node and multi node cluster.

Reply:
Hi @hubvu, any updates on this?

Reply:
@neoaggelos apologies for the delay, I have been working on other projects - I will work on the change this week and get back to you.

Reply:
Hi @hubvu, thank you. Please keep in mind that the 1.26 release is scheduled for next week (December 6th), so the deadline for new addons is the end of the week (December 2nd).

Please make sure to update the PR in time, otherwise it might be pushed back to the 1.27 release. Thank you

Reply:
Hi, please also make sure to rebase against the latest changes. Thanks a lot!

Reply:
@neoaggelos I have gone ahead and made the changes to the enable script so that it is a non-interactive install. I have also resolved the conflicts with the main branch.

Feel free to run the tests again and let me know if you need me to do anything else.

Thanks!